WordPress/Docs/WP/DeprecatedParametersStandard.xml

```diff
@@ -0,0 +1,19 @@
+<documentation title="Deprecated Parameters">
+    <standard>
+    <![CDATA[
+    Please refrain from using deprecated WordPress Parameters. 
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: use of non deprecated parameters.">
+        <![CDATA[
+get_the_author();
+        ]]>
+        </code>
+        <code title="Invalid: use of deprecated parameters.">
+        <![CDATA[
+get_the_author( $string ); // Deprecated WP 2.1
```

Comment:
Have changed this. Don't see the bold in the test though.

Reply:
The emphasis (not bold) may not show up in the command line output (`--generator=text`), but it may show up in `--generator=html` or `--generator=markdown`, etc.